ocs_ci/utility/utils.py

```diff
@@ -2574,3 +2575,24 @@ def check_for_rhcos_images(url):
     """
     r = requests.head(url)
     return r.status_code == requests.codes.ok
+
+
+def download_file_from_git_repo(git_repo_url, path_to_file_in_git, filename):
+    """
+    Download a file from a specified git repository
+
+    Args:
+        git_repo_url (str): The git repository url
+        path_to_file_in_git (str): Path to the file to download
+            in git repository
+        filename (str): Name of the file to write the download to
+
+    """
+    log.debug(
+        f"Download file '{path_to_file_in_git}' from "
+        f"git repository {git_repo_url} to local file '{filename}'."
+    )
+    temp_dir = mkdtemp()
+    git.Repo.clone_from(git_repo_url, temp_dir, branch='master', depth=1)
```

Comment:
We have function for cloning git repo: `clone_repo` in utils:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L1408

This does not require any requirements. What about use rather this one?

Instead of have master branch hardcoded here, please make it parametrized from  download_file_from_git_repo function.  So if someone will need to reuse this function for different branch it will be easier.